mediathread/templates/base.html

```
@@ -7,10 +7,12 @@
     	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
     	<title>Mediathread {% block title %}&mdash; Switch Course{% endblock %}</title>
 
+        {% if not settings.DEBUG %}
```

What happens if this script isn't loaded on the pages where it's expecting the analytics lib to be loaded? does it just silently ignore those calls?